SimPEG/Directives.py

```diff
@@ -216,7 +216,7 @@ def endIter(self):
         # Save the data.
         ms = self.reg.Ws * ( self.reg.mapping * (self.invProb.curModel - self.reg.mref) )
```

Comment:
This should talk directly to the regularization class - see #295 regarding modularizing the regularization so that each component is accessible